src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetStartup.cs

```diff
@@ -14,18 +14,7 @@ public static class AspNetStartup
         /// </summary>
         public static void Register()
         {
-            Tracer.Instance = new Tracer(
-                settings: null,
-                agentWriter: null,
-                sampler: null,
-                scopeManager: new AspNetScopeManager(),
```

Comment:
In my changes, I did not create a new `AspNetScopeManager`. I'm not familiar with it enough to know if this is a valid change or not

Reply:
It is necessary IIRC, because the call context doesn't flow the same as the http context. Traces will get crossed otherwise. I will get the load test hooked up for each of the major aspnet/aspnetcore frameworks and check for you.

Reply:
It's possible that by delaying the initialization it might flow correctly.....? We'll see.

Reply:
Resolving because I've reverted this change.